### PR TITLE
Add (Get and Add) Device Fingerprint cmdlet

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

File: PowerArubaCP/Public/DeviceFingerprint.ps1 [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+#
+
+function Add-ArubaCPDeviceFingerprint {
+
+    <#
+        .SYNOPSIS
+        Add a Device Fingerprint on ClearPass
+
+        .DESCRIPTION
+        Add an Device Fingerprint with mac address, hostname, ip (address) and device info (Category, Name, Familly)
```
Comment (alagoutte) [inline PowerArubaCP/Public/DeviceFingerprint.ps1:14]: an -> a

Comment (alagoutte) [reply PowerArubaCP/Public/DeviceFingerprint.ps1:14]: familly -> family

File: PowerArubaCP/Public/DeviceFingerprint.ps1 [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        }
+
+        if ( $PsBoundParameters.ContainsKey('device_family') ) {
+            $_device | add-member -name "family" -membertype NoteProperty -Value $device_family
+        }
+
+        $_dfp | add-member -name "device" -membertype NoteProperty -Value $_device
+
+        $dfp = Invoke-ArubaCPRestMethod -method "POST" -body $_dfp -uri $uri -connection $connection
+        $dfp
```
Comment (CedricMoreau) [inline PowerArubaCP/Public/DeviceFingerprint.ps1:86]: add a blank line ?

File: PowerArubaCP/Public/DeviceFingerprint.ps1 [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+        .EXAMPLE
+        Get-ArubaCPDeviceFingerprint -ip_address 192.0.2.1
+
+        Get Device FingerPrint about Endpoint 192.0.2.1 Aruba on the ClearPass
+
+    #>
+
+    Param(
+        [Parameter (ParameterSetName = "mac_address", Mandatory = $true)]
```
Comment (CedricMoreau) [inline PowerArubaCP/Public/DeviceFingerprint.ps1:120]: the two parameter are mandatory ? so the examples need to be updated

**Pull request merged**